manifests/manage_keys.pp

```diff
@@ -13,7 +14,7 @@
   ssh_authorized_key { $key_title:
     ensure => present,
     user   => $user,
-    name   => $key_name,
+    name   => "${user}_${key_name}",
```

Comment:
Is this actually necessary? Think rest of these changes should fix this problem without this?